client/layout/guided-tours/config.js

```diff
@@ -6,113 +6,116 @@
 import React from 'react';
 import i18n from 'i18n-calypso';
 
-function get( site ) {
-	return {
-		init: {
-			text: i18n.translate( "{{strong}}Need a hand?{{/strong}} We'd love to show you around the place, and give you some ideas for what to do next.", {
-				components: {
-					strong: <strong />,
-				}
-			} ),
-			type: 'FirstStep',
-			placement: 'right',
-			next: 'my-sites',
+function get( tour = 'main' ) {
+	const tours = {
+		main: {
+			version: '201600601',
+			init: {
+				text: i18n.translate( "{{strong}}Need a hand?{{/strong}} We'd love to show you around the place, and give you some ideas for what to do next.", {
+					components: {
+						strong: <strong />,
+					}
+				} ),
+				type: 'FirstStep',
+				placement: 'right',
+				next: 'my-sites',
+			},
+			'my-sites': {
+				target: 'my-sites',
+				arrow: 'top-left',
+				type: 'ActionStep',
+				icon: 'my-sites',
+				placement: 'below',
+				text: i18n.translate( "{{strong}}First things first.{{/strong}} Up here, you'll find tools for managing your site's content and design.", {
+					components: {
+						strong: <strong />,
+					}
+				} ),
+				next: 'sidebar',
+			},
+			sidebar: {
+				text: i18n.translate( 'This menu lets you navigate around, and will adapt to give you the tools you need when you need them.' ),
+				type: 'BasicStep',
+				target: 'sidebar',
+				arrow: 'left-middle',
+				placement: 'beside',
+				next: 'click-preview',
+			},
+			'click-preview': {
+				target: 'site-card-preview',
+				arrow: 'top-left',
+				type: 'ActionStep',
+				iconText: i18n.translate( "your site's name", {
+					context: "Click your site's name to continue.",
+				} ),
+				placement: 'below',
+				showInContext: site => site && site.is_previewable,
```

Comment:
Down the road, will these be a function of `state` or even `context`, rather than the more specific `site`?